docs/setup_vscode_wsl.md

```diff
@@ -244,36 +244,36 @@ $ g++ -g main.cpp -o main.exe
 <div class="primer-spec-callout warning" markdown="1">
 **Pitfall:** Make sure you're in WSL mode.
 
-<img src="images/vscode069.png" width="768px">
+<img src="images/vscode_wsl_069.png" width="768px">
 
 If you accidentally open VS Code from Windows mode, click on the green icon in the lower left hand corner and then select "Reopen Folder in WSL".
 
-<img src="images/vscode070.png" width="720">
+<img src="images/vscode_wsl_070.png" width="720">
 </div>
 
 Select the file you would like to run.  Navigate to the debugging pane.
 
-<img src="images/vscode030w.png" width="768px" />
+<img src="images/vscode_wsl_030w.png" width="768px" />
```

Comment:
Should be `vscode_wsl_030.png`.

Reply:
Fixed in [edefb88](https://github.com/eecs280staff/tutorials/pull/154/commits/edefb88e37bde4d507cd59389403addf3e5d4856).